helix-term/src/keymap/default.rs

```diff
@@ -211,6 +211,7 @@ pub fn default() -> HashMap<Mode, KeyTrie> {
 
         // move under <space>c
         "C-c" => toggle_comments,
+        "C-y" => insert_char_interactive,
```

Comment:
Unless there's a compelling keybinding for this let's leave it out of the default keymap for now. If we introduce a new command that fits well with `C-y` in the future we'd need a break muscle memory to move this.